lib/src/component_declaration/component_base.dart

```diff
@@ -31,6 +33,7 @@ import 'package:over_react/src/component_declaration/component_type_checking.dar
 import 'package:over_react/src/util/ddc_emulated_function_name_bug.dart' as ddc_emulated_function_name_bug;
 import 'package:react/react.dart' as react;
 import 'package:react/react_client.dart';
+import 'package:w_common/disposable.dart';
```

Comment:
I have created [RAP-2130](https://jira.atl.workiva.net/browse/RAP-2130) to address this.